ui/dashboards/src/components/Panel/Panel.stories.tsx

```diff
@@ -0,0 +1,111 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Thank you for adding stories for this! It's cool to see how a panel works by itself like this. 😄

Reply:
It was cool and easy to see what providers a panel needs! The only issue I noticed is when you click "Show code" on the docs page, the sample code is not very helpful.